proto/regen/curation/v1/tx.proto

```diff
@@ -0,0 +1,132 @@
+syntax = "proto3";
+package regen.curation.v1;
+
+option go_package = "github.com/regen-network/regen-ledger/x/curation";
+
+// Msg is the regen.curation.v1 Msg service.
+service Msg {
+    // CreateTagSet creates a new tag set with the provided metadata and returns an auto-generated tag set ID.
+    rpc CreateTagSet(MsgCreateTagSet) returns (MsgCreateTagSetResponse) {}
```

Comment:
I would rename `MsgCreateTagSetRequest` to `MsgCreateTagSet`. It applies to other rpc methods as well., e.g. `MsgCreateTagRequest` to `MsgCreateTag`...etc

Reply:
see https://github.com/regen-network/regen-ledger/pull/395#issuecomment-870608008